issue_drafts/issue_1.md

```diff
@@ -1,3 +1,145 @@
 # Issue 1: Introduction to Snakefiles
 
+Pipeline steps: Download the temperature predictions.
+Concepts learned: Snakefile structure, the `script` directive, basic `snakemake` commands 
+
+## Tutorial Overview
+
+Welcome to the USGS Data Science Practitioner's Snakemake tutorial!
+This tutorial consists of several issues.
+Each issue will guide you in building a piece of a Snakemake data pipeline.
+This pipeline will download, aggregate, and visualize predictions of lake temperatures at multiple depths.
+At the same time, each issue will introduce you to new Snakemake concepts.
+
+Most of the Python code for the pipeline is already written.
+It's organized into three phases.
+- `1_fetch` downloads the predictions from this data release on ScienceBase: https://www.sciencebase.gov/catalog/item/5e5d0bb9e4b01d50924f2b36 and unzips the files.
+- `2_process` computes the mean prediction on each day of the year for a selection of lakes and aggregates those means into a single .csv file.
+- `3_plot` plots the mean temperatures on each day of the year for each lake at a selection of depths.
+
+You can find the code organized by phase in three folders.
+Within each folder are all the Python scripts needed to execute the tasks of that phase. 
+If we wanted to carry out these tasks using the scripts as they are, we could execute each script in order.
+Alternatively, we can use a Snakemake pipeline.
+Snakemake allows us to:
+
+- Automate the execution of the scripts
```

Comment:
What do you think of adding this bullet? I know you can always just write documentation to describe the order of your workflow and how it all fits together, but I think one of the benefits to using a pipelining tool is having this formally documented. 
```suggestion
- Describe the order in which scripts should run, and what the expected inputs/outputs of each step are
- Automate the execution of the scripts
```